doc/user/content/ops/troubleshooting.md

```diff
@@ -10,58 +10,120 @@ aliases:
 ---
 
 You can use the queries below for spot debugging, but it may also be
-helpful to make them more permanent tools by creating them as views for `SUBSCRIBE`ing or materialized views to read more efficiently. Note that the
-existence of these additional views may itself affect performance.
+helpful to monitor their output by `SUBSCRIBE`ing to their change stream, e.g.,
+by issuing a command `COPY (SUBSCRIBE (<query>) TO stdout`.
+Note that this additional monitoring may, however, itself affect performance.
+
+### Limitations on Introspection Sources and Views for Troubleshooting
+
+Importantly, the introspection sources and views used in the queries below
+should be considered subject to changes in the future without further notice.
+This is why these sources and views are all created in the `mz_internal`
+schema, and it is not possible to create higher-level views depending on them.
+
+Introspection sources are maintained by independently collecting internal logging
+information within each of the replicas of a cluster. Thus, in a multi-replica
+cluster, the queries below need to be directed to a specific replica by issuing
+the command `SET cluster_replica = <replica_name>`. Materialize also directly
+exposes replica-specific introspection sources by suffixing the respective catalog
+relation names with a replica number. For example, `mz_internal.mz_compute_frontiers_1`
+corresponds to the introspection source `mz_internal.mz_compute_frontiers` in the
+first replica of the cluster.
```

Comment:
Thanks, this is very helpful! Would you take a second look at the revised text?

Reply:
Note that I did not point to `mz_cluster_replicas` since I did not find it in the documentation. I think you already had this doc addition in your queue for another PR?

Reply:
Looks great now!

I'm adding documentation for `mz_cluster_replicas` in https://github.com/MaterializeInc/materialize/pull/15212. We should make sure to add the missing link to whichever PR will get merged second :)